proposals/mainnet2-upgrade-proposal/plan.md

```diff
@@ -0,0 +1,75 @@
+# Upgrade Procedure
+
+**Note**: It is assumed you are currently operating a full-node running v0.8.2 version of Akash.
+
+1. Verify you are currently running the correct version of Akash 
+```
+akashd version --long
+name: akash
+server_name: akashd
+client_name: akashctl
+version: 0.8.2
+commit: 107f71bcd2804196b9f3194f3528b4b66a662e06
+build_tags: netgo,ledger,mainnet
+go: go version go1.15.5 linux/amd64
+```
+
+2. Export existing state from akashnet-1:
+**Note**: It is recommended for validators and operators to take a full data snapshot at the export height before proceeding in case the upgrade does not go as planned or if not enough voting power comes online in a sufficient and agreed upon amount of time. In such a case, the chain will fallback to continue operating akashnet-1.
+
+Before exporting state via the following command, the akashd binary must be stopped.
+
+`$ akashd export --for-zero-height --height=<TBD> > akashnet_1_genesis_export.json`
+
+3. Verify the SHA256 of the (sorted) exported genesis file:
+```
+$ jq -S -c -M '' akashnet_1_genesis_export.json | shasum -a 256
+```
+
+4. Build v0.9.3 of Akash
```

Comment:
Should we make the version TBD here?

Also should we consider recommending the deb/rpm releases?

Reply:
Yes, we should add TBD. And yeah, we'll add both options (build from source and using direct binaries)